tnt-application/src/main/java/org/dcsa/tnt/controller/unofficial/EventSubscriptionDaoController.java

```diff
@@ -0,0 +1,31 @@
+package org.dcsa.tnt.controller.unofficial;
+
+import lombok.RequiredArgsConstructor;
+import org.dcsa.skernel.errors.exceptions.ConcreteRequestErrorMessageException;
+import org.dcsa.tnt.persistence.entity.EventSubscription;
+import org.dcsa.tnt.persistence.repository.EventSubscriptionRepository;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+/**
+ * Just for testing.
+ */
+@RestController
+@RequiredArgsConstructor
+public class EventSubscriptionDaoController {
+  private final EventSubscriptionRepository eventSubscriptionRepository;
+
+  @Profile("test")
```

Comment:
This can be moved to the class level. Considering the whole class is just for testing purpose.

Reply:
Done, and also for the other unofficial controller